bin/kafka-streams-application-reset.sh

```diff
@@ -18,4 +18,4 @@ if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
 
-exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.tools.StreamsResetter "$@"
+exec $(dirname $0)/kafka-run-class.sh kafka.tools.StreamsResetter "$@"
```

Comment:
There are a couple of situations where the fully qualified name matters:

1. Users who invoke the tool from Java code. Perhaps from tests or some orchestration code in a development environment.

2. Users who invoke the tool via `kafka-run-class`. This is a bit of an anti-pattern, but our documentation uses that pattern for other tools at times (http://kafka.apache.org/documentation.html)

I'm OK with saying that neither of the usages above are valid in 0.10.0.1, that the class is not public API and that people have to use the `kafka-streams-application-reset.sh` if they expect compatibility going forward. If we go in that direction, the javadoc should make this clear.

Regarding who makes the final decision, that's a good question. I'd like to hear @guozhangwang's opinion on this as he was the committer who reviewed the PR.